### PR TITLE
Update css select (updated to latest develop)

### PR DESCRIPTION
**Pull request by Jyrno42**
This is a version of #1612 which has been updated to work with current develop branch. Hopefully @ereddrex does not mind me cleaning it up as I needed this quicker. I am happy to close this one in case the original author wants to continue with their work themselves.

Original PR info:

```
# Summary

This package uses an older version of css-select that is dependent upon a css-what package that contains high priority security violations. I am updating dependencies of this package newer, non-violating packages.

# Test Plan

Ran existing tests and all were successful.
```

### CODE REVIEW

**Review by MadeinFrance — APPROVED** [review]

**Comment by jphpsf** [comment]
Hi @MadeinFrance @brentvatne - thank you for approving this pull request. Is there any plan to merge this PR and create an updated release? Thank you very much :)

**Comment by ericanderson** [comment]
@jphpsf I wouldn't want to merge this either. It changes far more than just css-select. I would prefer this: https://github.com/react-native-svg/react-native-svg/pull/1692

File: package.json [hunk 1]
```diff
@@ -59,36 +59,40 @@
     "react-native": ">=0.50.0"
   },
   "dependencies": {
-    "css-select": "^2.1.0",
-    "css-tree": "^1.0.0-alpha.39"
+    "@babel/compat-data": "^7.14.7",
```
Comment (Havunen) [inline package.json:62]: shouldn't babel be devDependency ?

**Comment by jphpsf** [comment]
@ericanderson That's a very good point, thank you for submitting PR #1692

**Comment by ericanderson** [comment]
@jphpsf your other changes may be valid but it makes it harder to merge for the more urgent thing.

**Comment by jphpsf** [comment]
@ericanderson Not a problem! By the way, it's not my PR or changes, I was just checking on the PR status since I was looking for that fix :)

**Comment by WoLewicki** [comment]
Since https://github.com/react-native-svg/react-native-svg/pull/1692 has been merged, I will close it since it concerns the same thing. Feel free to comment here if I am wrong and we can reopen it then.

**Pull request closed without merging**